roles/kubernetes-apps/ansible/defaults/main.yml

```diff
@@ -41,9 +41,7 @@ netchecker_server_memory_requests: 64M
 # Dashboard
 dashboard_enabled: true
 dashboard_image_repo: gcr.io/google_containers/kubernetes-dashboard-amd64
-dashboard_image_tag: v1.8.1
-dashboard_init_image_repo: gcr.io/google_containers/kubernetes-dashboard-init-amd64
```

Comment:
Why drop the init container?  I dont have too much experience with the dashboard but just curious why it's getting dropped.

Reply:
It is not used anywhere else and I have no idea what is it for

Reply:
It was replaced by `auto-generate-certificates` argument added directly to dashboard container and this dependency was dropped in `v1.8.0` release (https://github.com/kubernetes/dashboard/releases/tag/v1.8.0).